idsvr/README.md

```diff
@@ -42,78 +42,80 @@ Tutorials on using the chart and more are available in the [Curity Resource libr
 In the table below you can find information about the parameters that are configurable in this chart.
 
 
-Parameter | Description                                                                                                                          | Default
```

Comment:
Looks like you inadvertently changed the whitespace for this entire table. Please correct it so `git blame` shows you only changing the lines you need to change.

Reply:
If I don't format the table, it was unclear to me what the spacing standard of this table is.  Different rows had different numbers of spaces, but the pipe characters didn't line up. If someone can explain to me what spacing I should use, I'm happy to back out this change (or at least make it a separate commit).